Makefile

```diff
@@ -128,8 +128,9 @@ container-push: container
 
 .PHONY: container-release
 container-release: container
+	# https://git-scm.com/docs/git-tag#Documentation/git-tag.txt---points-atltobjectgt
 	@docker tag $(DOCKER_REPO):$(VCS_BRANCH)-$(BUILD_DATE)-$(VERSION) $(DOCKER_REPO):$(VERSION)
-	docker push $(DOCKER_REPO):$(VERSION)
+	docker push $(DOCKER_REPO):$(strip $(shell [ -d .git ] && git tag --points-at HEAD))
 	docker push $(DOCKER_REPO):latest
```

Comment:
`$(DOCKER_REPO):latest` is not tagged in this make target. So if you do `make container-release` without first doing `make container-push`, you won't have a `latest` tag to push

Reply:
It's tagged a `container` stage which is a dependency of this action.